tests/dates/test_DatetimeSinusoidCalculator.py

```diff
@@ -204,20 +219,22 @@ def test_valid_units_value_error(self):
                 24,
             )
 
-    def test_attributes(self, example_transformer):
-        """Test that the value passed for new_column_name and units are saved in attributes of the same name."""
-        ta.classes.test_object_attributes(
-            obj=example_transformer,
-            expected_attributes={
-                "columns": ["a"],
-                "units": "hour",
-                "period": 24,
-            },
-            msg="Attributes for DateDifferenceTransformer set in init",
-        )
 
+class TestFit(GenericFitTests):
+    """Generic tests for transformer.fit()"""
+
+    @classmethod
+    def setup_class(cls):
+        cls.transformer_name = "DatetimeSinusoidCalculator"
+
+
+class TestTransform(GenericTransformTests, DatetimeMixinTransformTests):
+    """Tests for BaseTwoColumnDateTransformer.transform."""
+
+    @classmethod
+    def setup_class(cls):
+        cls.transformer_name = "DatetimeSinusoidCalculator"
 
-class TestDatetimeSinusoidCalculatorTransform:
     @pytest.mark.parametrize(
```

Comment:
Think we can delete the date col check on line 248

Reply:
yep agree https://github.com/lvgig/tubular/pull/310/commits/b897d286e4b741ae8d65ebe850724d7d4729dd53